packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -30,6 +30,30 @@ import { Badge, BadgeProps } from "./Badge";
 import { Box } from "./Box";
 import { HtmlProps } from "./HtmlProps";
 import { useOdysseyDesignTokens } from "./OdysseyDesignTokensContext";
+import { type FeatureTestSelector } from "./test-selectors";
+
+export const TabsTestSelectors = {
+  feature: {
+    tabItem: {
+      selector: {
+        method: "ByRole",
```

Comment:
_(not a blocker, but perhaps for future considerations)_
In general, it would be good to get these selector methods from an object, like `SelectorMethod.ByRole`, for example.
We can even type it where it's something like:
`type SelectorMethodType = keyof typeof SelectorMethod`

My reasoning is to avoid any string errors, keep things DRY and control the types of selectors we support.

Reply:
That would work. It's already type-safe today with the `satisfies` method at the bottom:
![image](https://github.com/okta/odyssey/assets/97464104/995049b8-6949-4a47-9e2e-686206a96aed)

Not sure if that changes your suggestion or not.